README.md

```diff
@@ -26,7 +26,8 @@ This project also leverages the same ansible playbook internally for OCP deploym
 
 The automation needs to run from a system with internet access. This could be your laptop or a VM with public internet connectivity. This automation code has been tested on the following 64-bit Operating Systems:
 - Mac OSX (Darwin)
-- Linux (x86_64)
+- Linux (x86_64/ppc64le)
+- Windows(Cygwin, Git Bash)
```

Comment:
We do not have support for Windows yet. I need to work on that issue #66.

Can you please remove it from the commit.